docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -98,32 +121,40 @@ If the viewer doesn't have an invitation or it's been revoked, you won't be noti
 
 ### Revoke access for a viewer
 
-1. Click the sharing icon to the right of the dashboard title.
+1. Click the sharing icon in the dashboard header.
 1. Click the **Public dashboard** tab.
 1. Click **Revoke** on the viewer you'd like to revoke access for.
 
 Immediately, the viewer no longer has access to the public dashboard, nor can they use any existing one-time use links they may have.
 
 ### Reinvite a viewer
 
-1. Click the sharing icon to the right of the dashboard title.
+1. Click the sharing icon in the dashboard header.
 1. Click the **Public dashboard** tab.
 1. Click **Resend** on the viewer you'd like to re-share the public dashboard with.
 
 The viewer will receive an email with a new one-time use link. This will invalidate all previously issued links for that viewer.
 
 ### Access limitations
 
-One-time use links use browser cookies, so when a viewer is granted access via one of these links, they will only have access through the browser they used to claim the link.
+One-time use links use browser cookies, so when a viewer is granted access through one of these links, they will only have access on the browser they used to claim the link.
 
 A single viewer cannot generate multiple valid one-time use links. When a new one-time use link is issued for a viewer, all previous ones are invalidated.
 
 If a Grafana user has read access to the parent dashboard, they can view the public dashboard without needing to have access granted.
 
+## Assess public dashboard usage
```

Comment:
Public dashboard insights are only available in the same versions as Dashboard insights (Enterprise, Cloud Pro and Cloud Advanced).